src/vs/workbench/contrib/positronKeybindings/browser/positronKeybindings.contribution.ts

```diff
@@ -0,0 +1,273 @@
+		// Create new R file
+		this._registrations.add(KeybindingsRegistry.registerKeybindingRule({
+			id: 'r.createNewFile',
+			weight: KeybindingWeight.WorkbenchContrib,
+			primary: KeyMod.CtrlCmd | KeyCode.KeyN
+		}));
+
+		// Go to/reveal definition
+		this._registrations.add(KeybindingsRegistry.registerKeybindingRule({
+			id: 'editor.action.revealDefinition',
+			weight: KeybindingWeight.WorkbenchContrib,
+			when: EditorContextKeys.editorTextFocus,
+			primary: KeyCode.F2
+		}));
+
+		// Focus "Source pane" (active editor group)
+		this._registrations.add(KeybindingsRegistry.registerKeybindingRule({
+			id: 'workbench.action.focusActiveEditorGroup',
+			weight: KeybindingWeight.WorkbenchContrib,
+			mac: {
+				primary: KeyMod.CtrlCmd | KeyCode.Digit1,
+				secondary: [KeyMod.WinCtrl | KeyCode.Digit1]
+			},
+			primary: KeyMod.CtrlCmd | KeyCode.Digit1
+		}));
+
+		// Focus "Console pane" (the Console)
+		this._registrations.add(KeybindingsRegistry.registerKeybindingRule({
+			id: 'workbench.action.positronConsole.focusConsole',
+			weight: KeybindingWeight.WorkbenchContrib,
+			mac: {
+				primary: KeyMod.CtrlCmd | KeyCode.Digit2,
+				secondary: [KeyMod.WinCtrl | KeyCode.Digit2]
+			},
+			primary: KeyMod.CtrlCmd | KeyCode.Digit2
+		}));
+
+		// Rename symbol
+		this._registrations.add(KeybindingsRegistry.registerKeybindingRule({
+			id: 'editor.action.rename',
+			weight: KeybindingWeight.WorkbenchContrib,
+			when: EditorContextKeys.editorTextFocus,
+			primary: KeyMod.CtrlCmd | KeyMod.Alt | KeyMod.Shift | KeyCode.KeyM
+		}));
+
+		// Comment line
+		this._registrations.add(KeybindingsRegistry.registerKeybindingRule({
+			id: 'editor.action.commentLine',
+			weight: KeybindingWeight.WorkbenchContrib,
+			when: EditorContextKeys.editorTextFocus,
+			primary: KeyMod.CtrlCmd | KeyMod.Shift | KeyCode.KeyC
+		}));
+
+		// Show all symbols
+		this._registrations.add(KeybindingsRegistry.registerKeybindingRule({
+			id: 'workbench.action.showAllSymbols',
+			weight: KeybindingWeight.WorkbenchContrib,
+			mac: {
+				primary: KeyMod.CtrlCmd | KeyCode.Period,
+				secondary: [KeyMod.WinCtrl | KeyCode.Period]
+			},
+			primary: KeyMod.CtrlCmd | KeyCode.Period
+		}));
+
+		// Open keybindings
+		this._registrations.add(KeybindingsRegistry.registerKeybindingRule({
+			id: 'workbench.action.openGlobalKeybindings',
+			weight: KeybindingWeight.WorkbenchContrib,
+			primary: KeyMod.Shift | KeyMod.Alt | KeyCode.KeyK
+		}));
+
+		// Insert code cell
+		this._registrations.add(KeybindingsRegistry.registerKeybindingRule({
+			id: 'quarto.insertCodeCell',
+			weight: KeybindingWeight.WorkbenchContrib,
+			when: ContextKeyExpr.and(
+				EditorContextKeys.editorTextFocus,
+				ContextKeyExpr.equals(EditorContextKeys.languageId.key, 'quarto')),
+			primary: KeyMod.CtrlCmd | KeyMod.Alt | KeyCode.KeyI
+		}));
+
+		// Run current
+
+		this._registrations.add(KeybindingsRegistry.registerKeybindingRule({
+			id: 'quarto.runCurrent',
```

Comment:
should this have a mac clause for KeyMod.WinCtrl?
https://github.com/posit-dev/positron/blob/64988dfaeaa5de9219cb777569c2e8631aefbe0e/extensions/positron-rstudio-keymap/package.json#L125

Reply:
Good question ... I omitted this intentionally b/c I think this particular binding is typically just run with Cmd+Enter on macOS. I'll add it back if folks say they want it for Ctrl too!